daemon/images/image_prune.go

```diff
@@ -119,7 +119,7 @@ deleteImagesLoop:
 
 			if shouldDelete {
 				for _, ref := range refs {
-					imgDel, err := i.ImageDelete(ref.String(), false, true)
+					imgDel, err := i.ImageDelete(ctx, ref.String(), false, true)
 					if imageDeleteFailed(ref.String(), err) {
```

Comment:
Looks like `imageDeleteFailed()` needs to be updated so it doesn't log a spurious warning when the context is canceled.